cmd/rpctest/rpctest/bench_ethgetblockbyhash.go

```diff
@@ -0,0 +1,115 @@
+package rpctest
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+// BenchEthGetBlockByHash compares response of Erigon with Geth
+// but also can be used for comparing RPCDaemon with Geth or infura
+// parameters:
+// needCompare - if false - doesn't call Erigon and doesn't compare responses
+//
+//			    false value - to generate vegeta files, it's faster but we can generate vegeta files for Geth and Erigon
+//	                 recordFile stores all eth_call returned with success
+//	                 errorFile stores information when erigon and geth doesn't return same data
+func BenchEthGetBlockByHash(erigonURL, gethURL string, needCompare, latest bool, blockFrom, blockTo uint64, recordFile string, errorFile string) {
+	setRoutes(erigonURL, gethURL)
+	var client = &http.Client{
+		Timeout: time.Second * 600,
+	}
+
+	var rec *bufio.Writer
+	var errs *bufio.Writer
+	var resultsCh chan CallResult = nil
+	var nBlocks = 0
+
+	if errorFile != "" {
+		f, err := os.Create(errorFile)
+		if err != nil {
+			fmt.Printf("Cannot create file %s for errorFile: %v\n", errorFile, err)
+			return
+		}
+		defer f.Close()
```

Comment:
optionally - can add `Sync` before `Close` - if can't afford loosing this data at power-off.

Reply:
ok

Reply:
oops. defers are executed in reverse order